floodviz/static/js/map.js

```diff
@@ -236,6 +232,34 @@
 				svg.select('#map-select-box').remove();
 			};
 
+			var make_sites_bland = function () {
+				Object.keys(state.gages).forEach(function (key) {
+					const g = state.gages[key];
+					var style = 'gage-point';
+					if (g.accent) {
+						style += '-accent'
+					}
+					if (g.bland) {
+						style += '-bland';
+					}
+					else {
+						g.bland = true;
+					}
+					d3.select('#map' + key).attr('class', style);
```

Comment:
I like the style class appending here! But the function name doesn't necessarily represent the actual functionality well if logic has the possibility to accent and not make bland here. 

`if (g.accent) {style += '-accent'}` is a part of make_sites_normal. Why use it here as well? Also do we want to be setting g.bland to true if it was already false? Is this preemptive?